### ISSUE TITLE
Using with Kdyby/Console

### ISSUE THREAD

**Issue by Gappa** [contributte/console-extra#13]
Hi,

maybe some people will want to start using this package while still using Kdyby/Console. I was using modette/cache-cleaner before (which is now abandoned), which also required some "hacking" :) 

However, this package requires a bit different approach (maybe I'm just doing it wrong, I also don't want to edit other package's files). This is what I did to make `AdvancedCache\CacheCleanCommand` to work with Kdyby/Console:

``` yaml
services:
	 localFsCleaner: Contributte\Console\Extra\Cache\Cleaners\LocalFilesystemCleaner([%tempDir%/cache/])

console:
	commands:
		- Contributte\Console\Extra\Command\AdvancedCache\CacheCleanCommand([@localFsCleaner])
``` 

And voilá:
```
Available commands:
	help                                  Displays help for a command
	list                                  Lists commands
	contributte
		contributte:cache:clean            Clean cache 
```

No need to register the extension itself (which is questionable? :)

So, I was thinking of adding a guide to the docs for users interested in using this package who are still "vendor-locked" in Kdyby/Console. Is it a good idea? :)

**Comment by mabar** [comment]
Send a PR please. Copy style from [here](https://github.com/contributte/event-dispatcher/blob/master/.docs/README.md#kdyby-compatibility)

Extensions are just a shortcut. Majority of DI extensions just register services into DI.

You can also add tag `kdyby.console.command` to an existing service registered by extension.

And sorry for so fast abandonment of modette. We found that our plans are very similar so I merged everything with contributte. Modette will be a modular application building system instead.

**Comment by Gappa** [comment]
Ok, I'll try. But first I'll have to test different approaches a bit, just to be sure :)

**Comment by mabar** [comment]
Universal way :)

```yaml
decorator:
	Symfony\Component\Console\Command\Command:
		tags: [kdyby.console.command]
```

**Comment by Gappa** [comment]
Sent the PR :)

Regarding the abandonment of modette - well, it's a good thing to consolidate/merge packages & efforts into larger ones, that way they will get to more users and stay maintaned for longer :)

**Comment by mabar** [comment]
Closed by #14

**Comment by f3l1x** [comment]
Thanks @Gappa
